Add guidance to include contextualizing information in descriptions (#648)
grafana/writers-toolkit: docs/sources/write/front-matter/index.md

Use `description` to provide the short description of the topic to search engines, including the search engine used in the Grafana documentation site.
The description is also displayed on social media, such as Twitter, to provide a clue to users about the page contents.

> Your description should include contextual information like the product name because the reader isn't on the Grafana website.

Reviewer: ```suggestion
Because the reader isn't on the Grafana website, include contextual information in your description such as product name.
```

Reviewer 2: or maybe one more tweak? `Since the reader isn't on the Grafana website, your description should include contextual information, such as the product name.`  

(I hate starting a sentence with `Because` even though it's correct in this context :-) )

Author: ```suggestion
Since the reader isn't on the Grafana website, your description should include contextual information, such as the product name.
```
